ClangSharp/ClangSharp.csproj

```diff
@@ -3,6 +3,19 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>ClangSharp are strongly-typed safe Clang bindings written in C# for .NET and Mono, tested on Linux and Windows. ClangSharp is self-hosted and ClangSharp auto-generates itself parsing LLVM-C header files.</Description>
+    <Version>3.9.0</Version>
+    <PackageLicenseUrl>https://github.com/microsoft/ClangSharp/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <RepositoryUrl>https://github.com/microsoft/ClangSharp.git</RepositoryUrl>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageProjectUrl>http://www.clangsharp.org/</PackageProjectUrl>
   </PropertyGroup>
 
+  <ItemGroup>
+      <Content Include="ClangSharp.targets" Pack="true" PackagePath="build"/>
+      <Content Include="x64\libclang.dll" Pack="true" PackagePath="content\x64"/>

```

Comment:
In general, checking in binary files to git repos is considered "bad practice" and you should look at doing something else instead (such as pulling down a nuget package or using git lfs).

This should probably be pulling down the existing `libclang` package (but an updated version containing the 8.0 binaries).

Reply:
You are right, please ignore my changes, I will try to reuse libclang.runtime public package (I miss it).
The version (3.9.0) not related to libclang, current ClangSharp package available on nuget has 3.8.0 version so I simple use next version.

Reply:
I think just using https://www.nuget.org/packages/Libclang/, which is published alongside ClangSharp by @mjsabby would be better.

That likely requires a new version to be published first, however.

Reply:
libclang uses 5.0 version, IMO it is too old, no?

Reply:
Right, the current sources are based on `libclang 8.0.0`. So, it likely needs a new version to be published first.

mjsabby is the publisher of that package, and also the maintainer for this repo, so he should be able to do that 😄

Reply:
so lets wait :)